src/components/Button.js

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ ButtonName }) => (
-  <button type="button">
+const Button = ({ ButtonName, color, wide }) => (
+  <button type="button" className={`${color ? 'lightgray' : 'orange'} ${wide ? 'large' : 'small'}`}>
     {ButtonName}
   </button>
 );
 
+Button.defaultProps = {
+  color: false,
```

Comment:
opps!
`color` should be a string not `boolean`, and it should be set to orange as a default value.

![image](https://user-images.githubusercontent.com/31897434/95397170-72744200-08fa-11eb-9d73-2f5d25cdb66f.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896093/